src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginConstants.cs

```diff
@@ -7,4 +8,5 @@ public static class PipelineArtifactPluginConstants
         public static readonly Guid PublishPipelineArtifactTaskId = new Guid("ECDC45F6-832D-4AD9-B52B-EE49E94659BE");
         public static readonly Guid DownloadPipelineArtifactTaskId = new Guid("61F2A582-95AE-4948-B34D-A1B3C4F6A737");
     }
+       
 }
```

Comment:
revert this file as the changes are not required

Reply:
This is still not resolved, the new using statement isn't needed either. This file shouldn't appear as part of the list of changes.

---
In reply to: [291395680](https://github.com/microsoft/azure-pipelines-agent/pull/2279#discussion_r291395680) [](ancestors = 291395680)